example.ipynb

```diff
@@ -213,1997 +404,504 @@
-      "                  'derived_from_specimen': '401ac98d-44ef-4267-836c-14e481f0b517',\n",
-      "                  'derived_from_subject': 'TCGA-3P-A9WA',\n",
-      "                  'id': '92bb4159-c7d0-472a-816a-fb26672fc8c6',\n",
-      "                  'identifier': [ { 'system': 'GDC',\n",
-      "                                    'value': '92bb4159-c7d0-472a-816a-fb26672fc8c6'}],\n",
-      "                  'primary_disease_type': 'Cystic, Mucinous and Serous '\n",
-      "                                          'Neoplasms',\n",
-      "                  'source_material_type': 'Blood Derived Normal',\n",
-      "                  'specimen_type': 'aliquot'},\n",
-      "                { 'File': [ { 'associated_project': ['TCGA-OV'],\n",
-      "                              'byte_size': 216647924,\n",
-      "                              'checksum': '431606691f638bb07d9028e6605539c7',\n",
-      "                              'data_category': 'Simple Nucleotide Variation',\n",
-      "                              'data_type': 'Aggregated Somatic Mutation',\n",
-      "                              'file_format': None,\n",
-      "                              'gcs_path': 'gs://gdc-tcga-phs000178-controlled/04da990e-67a3-4ead-ab08-448c7118624c/TCGA.OV.varscan.04da990e-67a3-4ead-ab08-448c7118624c.DR-10.0.protected.maf.gz',\n",
-      "                              'id': '04da990e-67a3-4ead-ab08-448c7118624c',\n",
-      "                              'identifier': [ { 'system': 'GDC',\n",
-      "                                                'value': '04da990e-67a3-4ead-ab08-448c7118624c'}],\n",
-      "                              'label': 'TCGA.OV.varscan.04da990e-67a3-4ead-ab08-448c7118624c.DR-10.0.protected.maf.gz',\n",
-      "                              'type': None},\n"
+      "\n",
+      "Query: SELECT * FROM gdc-bq-sample.cda_mvp.v3, UNNEST(ResearchSubject) AS _ResearchSubject, UNNEST(_ResearchSubject.Specimen) AS _Specimen, UNNEST(_ResearchSubject.Diagnosis) AS _Diagnosis, UNNEST(_Specimen.identifier) AS _identifier WHERE ((_Specimen.primary_disease_type = 'Nevi and Melanomas') AND ((_Diagnosis.age_at_diagnosis < 30*365) AND (_identifier.system = 'GDC')))\n",
+      "Offset: 0\n",
+      "Limit: 1000\n",
+      "Count: 647\n",
+      "More pages: No\n",
+      "\n"
      ]
-    },
+    }
+   ],
+   "source": [
+    "q1 = Q('ResearchSubject.Specimen.primary_disease_type = \"Nevi and Melanomas\"')\n",
+    "q2 = Q('ResearchSubject.Diagnosis.age_at_diagnosis < 30*365')\n",
+    "q3 = Q('ResearchSubject.Specimen.identifier.system = \"GDC\"')\n",
+    "\n",
+    "q = q1.And(q2.And(q3))\n",
+    "r = q.run()\n",
+    "\n",
+    "print(r)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "psychological-revolution",
+   "metadata": {},
+   "source": [
+    "By inspecting the results, we can see that all 577 patients returned in the initial query are coming from the GDC.\n",
```

Comment:
I think we should avoid using a number here since this will change in the future.

"By comparing the `Count` value the two results we can see that all the patients returned in the initial query are coming from the GDC.\n",